graveler/sstable/writer.go

```diff
@@ -106,6 +116,14 @@ func (dw *DiskWriter) Close() (*committed.WriteResult, error) {
 	tableHash := dw.hash.Sum(nil)
 	sstableID := hex.EncodeToString(tableHash)
 
+	// Prepare metadata properties for Close to write.  The map was already set in the
+	// sstable.Writer constructor and cannot be changed, but we can replace its values
+	// before writing it out.
+	dw.AddMetadata("first_key", dw.first)
```

Comment:
perhaps we should align these values with our canonical names for these (i.e. `min_key`, `max_key` and whatever it is we're calling `num_records` which I believe is currently `count` (??)

Reply:
Done.  Although we _also_ call them "first" and "last" in `WriteResult`, just a few lines down.  Not cleaning that up right now, but metadata tags are forever.